### DISCUSSION TITLE
Reports of degraded availability/performance

### DISCUSSION THREAD

**Discussion by mikhail-tokarev** [codemagic-ci-cd/codemagic-docs#1581] [Announcements]
Codemagic is experiencing degraded performance. We are still investigating and will provide an update when we have one.

**Comment by mikhail-tokarev** [comment]
We have deployed a fix and are monitoring closely.

**Reply by fatherOfLegends** [reply]
Builds still failing on preparing build machine.

**Comment by mikhail-tokarev** [comment]
This incident has been resolved.
